sdk/include/opentelemetry/sdk/logs/logger_provider.h

```diff
@@ -19,6 +19,7 @@
 #include <memory>
 #include <mutex>
 #include <string>
+#include <unordered_map>
```

Comment:
This is only line in file's diff so not how this include for unordered_map could be used?

Reply:
I had actually not tested compiling it before :) The unordered_map is need for the list of Loggers stored by the loggerProvider